### PR TITLE
Sle 12 sp1

### PR DESCRIPTION
**Pull request by keichwa**

### CODE REVIEW

File: Rakefile [hunk 1]
```diff
@@ -3,7 +3,7 @@ require "yast/rake"
 Yast::Tasks.configuration do |conf|
```
Comment (jreidinger) [inline Rakefile]: this is wrong, please use Rakefile from master. ( this rakefile will submit to SLE12 maintenance update )

**Comment by jreidinger** [comment]
please fix conflict and issue mentioned above

**Comment by jreidinger** [comment]
@keichwa I fix conflict and Rakefile, please check if it is correct, so I can merge it.

**Comment by jreidinger** [comment]
@keichwa ping

**Comment by mvidner** [comment]
I don't understand this PR. What is it supposed to achieve? The referenced bug is a dummy :-(

**Comment by coolo** [comment]
update translations?

**Pull request merged**